src/pages/components/Footer/Footer.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react'
+import {Link} from 'react-router-dom'
+import "../../../Styles/common.scss"
+import "../../../Styles/reset.scss"
+import "./Footer.scss";
+
+export class Footer extends Component {
+  render() {
+    return (
+      <footer className="mgm-footer">
```

Comment:
이제는 하나의 프로젝트를 각각 기능에 따라 나누어서 구현하고 있기 때문에 최상단에 이니셜 이름이 들어갈 필요없이 class명과 동일하게 적어주시면 됩니다 :) 
```suggestion
      <footer className="Footer">
```

Reply:
넵, 이니셜 제거 했습니다